modules/core/variables.tf

```diff
@@ -305,13 +305,87 @@ variable "vault_user_data" {
   default = ""
 }
 
+# --------------------------------------------------------------------------------------------------
+# Internal LB Variables
+# --------------------------------------------------------------------------------------------------
+
 variable "internal_lb_name" {
   description = "Name of the internal load balancer"
   default     = "internal"
 }
 
-variable "deregistration_delay" {
-  description = "Time before an unhealthy Elastic Load Balancer target becomes removed"
+variable "nomad_server_lb_deregistration_delay" {
+  description = "The time to wait for in-flight requests to complete while deregistering a target. During this time, the state of the target is draining."
+  default     = 30
+}
+
+variable "nomad_server_lb_healthy_threshold" {
+  description = "The number of consecutive health checks successes required before considering an unhealthy target healthy (2-10)."
+  default     = 2
+}
```

Comment:
The value has been changed from 5 to 2 after refactoring into a variable, is this intentional?

Reply:
Yes. 5 is too long for a service recovery.

Reply:
Okay noted